locksmith/src/utils/image.ts

```diff
@@ -0,0 +1,14 @@
+export const imageToBase64 = async (url: string) => {
+  const response = await fetch(url, {
```

Comment:
Why are we fetching the image? It wastes resources because this image is already stored here, Let's just retrieve it.